specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4229,6 +4229,18 @@
             "description": "Type: string (or Expression with resultType string)."
           }
         },
+        "newClusterDriverNodeType": {
+          "type": "object",
+          "description": "The driver node type for the new cluster. Type: string (or Expression with resultType string)."
+        },
+        "newClusterInitScripts": {
+          "type": "object",
+          "description": "User-defined initialization scripts for the new cluster. Type: array of strings (or Expression with resultType array of strings)."
```

Comment:
similar mismatch here... if the type is array of string, shouldnt type reflect that .. it says object right now

Reply:
same for the next boolean property

Reply:
answered above